plugins/guests/windows/cap/change_host_name.rb

```diff
@@ -28,7 +28,9 @@ def self.change_host_name_and_wait(machine, name, sleep_timeout)
             error_key: :rename_computer_failed)
 
           # Don't continue until the machine has shutdown and rebooted
-          sleep(sleep_timeout)
+	  if machine.guest.capability?(:wait_for_reboot)
+            machine.guest.capability(:wait_for_reboot)
+	  end
```

Comment:
We should have an else clause here and just sleep. Perhaps leave the `sleep_timeout` param passed in and use that to fall back on?